plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -626,7 +600,7 @@ public WriteMapping toWriteMapping(ConnectorSession session, Type type)
             else {
                 dataType = "char(" + ((CharType) type).getLength() + " CHAR)";
             }
-            return WriteMapping.sliceMapping(dataType, charWriteFunction());
+            return WriteMapping.sliceMapping(dataType, oracleCharWriteFunction());
```

Comment:
seems orthogonal?

Reply:
I spotted that we use oracleCharWriteFunction on ColumnMapping but on write mapping we did not. I fixed that.

Reply:
That's fine. Let's make it a separate commit (like a prep one)